frontend/components/text2text/results/Text2TextList.vue

```diff
@@ -224,54 +207,53 @@ export default {
       if (this.sentencesOrigin === "Prediction") {
         return this.predictions;
       }
+      if (this.sentencesOrigin === "Preannotation") {
+        return this.predictions;
+      }
       return [];
     },
-    hasAnnotationAndPredictions() {
-      return this.predictions.length && this.annotations.length;
-    },
     allowValidation() {
       return (
         this.sentencesOrigin === "Prediction" ||
         (this.record.status !== "Validated" &&
           !!this.record?.sentenceForAnnotation)
       );
     },
-    selected() {
-      return this.record.selected;
-    },
     text2textClassifierActionButtons() {
       return [
         {
           id: "validate",
           name: "Validate",
-          allow: true,
+          allow: this.sentencesOrigin !== "Prediction",
           active: this.allowValidation,
         },
         {
           id: "discard",
           name: "Discard",
-          allow: true,
+          allow: this.sentencesOrigin !== "Prediction",
           active: this.record.status !== "Discarded",
         },
+        // {
+        //   id: "clear",
+        //   name: "Clear",
+        //   allow: this.sentencesOrigin !== "Prediction",
+        //   active: this.record.annotation || false,
+        // },
+        {
+          id: "reset",
+          name: "Reset",
+          allow: this.sentencesOrigin !== "Prediction",
+          active: this.record.status === "Edited",
+        },
```

Comment:
1/duplicate code : add a variable for the 2 test *this.sentencesOrigin !== "Prediction"* and a function for the test *"this.record.status !== "Discarded" or  "Edited"*
2/ delete comments
3/ create a functionfactory for construc the object ` {
          id: "discard",
          name: "Discard",
          allow: this.sentencesOrigin !== "Prediction",
          active: this.record.status !== "Discarded",
        }`